templates/admin/round_system_results.html.twig

```diff
@@ -0,0 +1,63 @@
+{% extends '@EasyAdmin/default/layout.html.twig' %}
+{% block content %}
+    <div class="container">
+        <h4>{{ tournament.title }}</h4>
+        <p>0 - чорні виграли, 1 - нічия, 2 - білі виграли</p>
```

Comment:
please implement translations for this text

Reply:
done